Homework2/Codewars.py

```diff
@@ -0,0 +1,38 @@
+# 1. In this kata you are required to, given a string, replace every letter
+# with its position in the alphabet.
+# If anything in the text isn't a letter, ignore it and don't return it.
+
+text = "Once upon a time..."
+alphabet = 'abcdefghijklmnopqrstuvwxyz'
```

Comment:
import string
string.ascii_lowercase will give this string

Reply:
thanks